gradle/dependencies.gradle

```diff
@@ -49,51 +49,51 @@ ext {
     javapoet: '1.11.1',
     jcache: '1.1.1',
     jcommander: '1.78',
-    jmh: '1.21',
+    jmh: '1.22',
     jsr330: '1',
     koloboke: '0.6.8',
     nullaway: '0.7.8',
     ohc: '0.6.1',
-    slf4j: '1.7.28',
+    slf4j: '1.7.29',
     tcache: '2.0.1',
     stream: '2.9.8',
     univocityParsers: '2.8.3',
-    ycsb: '0.15.0',
+    ycsb: '0.17.0',
     xz: '1.8',
   ]
   testVersions = [
     awaitility: '4.0.1',
     easymock: '4.0.2',
-    hamcrest: '2.1',
+    hamcrest: '2.2',
     jcacheTck: '1.1.1',
     jctools: '2.1.2',
     junit: '4.12',
     mockito: '3.1.0',
     paxExam: '4.13.1',
-    testng: '6.14.3',
+    testng: '7.0.0',
```

Comment:
I had memory issues with 7.0, so I am surprised it's working for you. I'd lean towards staying at 6.x for now to be safe.

See https://github.com/cbeust/testng/issues/2096 where I spent time trying to reduce the memory churn to compensate for TestNG's added bloat. They don't spool the test results to disk and instead hold it in memory, which doesn't scale for our test execution count (millions).